src/renderer/components/Execute.tsx

```diff
@@ -32,23 +50,27 @@ const Execute = () => {
         </div>
         <div>
           <h6>Selected Procedure</h6>
-          <ProcedureDropdown
-            activeDb={activeDb}
-            activeProcedure={activeProcedure}
-            setActiveProcedure={setActiveProcedure}
-            setCode={setCode}
-          />
-        </div>
-        <div className="code-wrapper">
-          <p className="procedure-code">{code}</p>
+          <div className="d-flex flex-direction-column">
+            <ProcedureDropdown
+              activeDb={activeDb}
+              activeProcedure={activeProcedure}
+              setActiveProcedure={setActiveProcedure}
+              setCode={setCode}
+            />
+            <Button variant="link" size="sm" onClick={() => setAlert(true)}>
```

Comment:
Maybe we should add some logic so that we don't `setAlert(true)` if no procedure is selected? Or maybe some sort of error: select a procedure popup?